narayana-release-process.sh

```diff
@@ -204,23 +199,27 @@ then
   exit
 fi
 git archive -o ../../narayana-full-$CURRENT-src.zip $CURRENT
-ant -f build-release-pkgs.xml -Drsync.host=${RSYNC_HOST} -Drsync.enabled=${RSYNC_ENABLED} -Dawestruct.executable="awestruct" all
-if [[ $? != 0 ]]
-then
-  echo 1>&2 COULD NOT BUILD Narayana RELEASE PKGS
-  exit
-fi
-cd -
+# build-release-pkgs.xml needs to be updated, update the website manually
+echo " narayana.io needs updating, please update the narayana.io repository (javadoc to unzip into api, project and product documentation and then upload to narayana.io"
```

Comment:
I think I would not know what to do based on the prompt provided here. But maybe [https://github.com/jbosstm/narayana.io needds ](https://github.com/jbosstm/narayana.io/blob/develop/README.md) should have the information you are speaking about in the bracketed bit of this and you can mainly refer to https://github.com/jbosstm/narayana.io/blob/develop/README.md here (specifying what parts to do)?

Reply:
Thanks for your suggestion. I will make sure all steps are well described in narayana.io README and narayana wiki. Let's keep this conversation as not resolved for now.

Reply:
Concerning this https://github.com/jbosstm/narayana.io/pull/187